pulp_ansible/app/tasks/utils.py

```diff
@@ -0,0 +1,52 @@
+import json
+
+from urllib.parse import parse_qs, urlencode, urlparse, urlunparse
+from aiohttp import ClientSession
+
+
+async def get_json_from_url(url):
+    """Asynchronously get the json from request."""
+    async with ClientSession() as session:
+        async with session.get(url) as response:
+            data = await response.json()
+            # NOTE(fao): remove the line below when the filter issue is addressed
```

Comment:
What needs to be done in this PR for this?

Reply:
the issue on galaxy collections api must be solved, it is for manually filter the namespace